src/components/ViewAdmin.vue

```diff
@@ -46,25 +46,39 @@ SPDX-License-Identifier: AGPL-3.0-or-later
 			<p>&nbsp;</p>
 			<p>{{ t('context_chat', 'Eligible files for indexing: {count}', {count: stats.eligible_files_count}) }}</p>
 			<p>{{ t('context_chat', 'Queued content update actions: {count}', {count: stats.queued_actions_count}) }}</p>
-			<p><a href="https://docs.nextcloud.com/server/latest/admin_manual/ai/app_context_chat.html">{{ t('context_chat', 'Official documentation') }}</a></p>
+			<h2>{{ t('context_chat', 'Context Chat Logs') }}</h2>
+			<NcButton :href="downloadURLNextcloudLogs">
+				{{ t('context_chat', 'Download Nextcloud logs') }}
+			</NcButton>
+			<NcButton :href="downloadURLDockerLogs">
+				{{ t('context_chat', 'Download Backend logs') }}
```

Comment:
some design comments:

1. let's convert the `h2` to `NcSettingsSection` same as the "Status" header above.
2. place the download buttons in the same line
3. add some gap between the buttons and below the buttons
4. maybe underline can be removed too but works either way

![image](https://github.com/user-attachments/assets/7e091b69-4ece-47b4-9edf-43cd743382a7)